src/charm_state.py

```diff
@@ -49,3 +40,21 @@ def github_org(self) -> str:
             str: github_org config.
         """
         return self._charm.config["github_org"]
+
+    @property
+    def github_exporter_command(self) -> str:
```

Comment:
Is this state or a constant?

Reply:
The port and container name is more like state than a command to execute, which you wouldn't expect to change

Reply:
State is related to information about the workload when it is operational, constants are things that we don't want to embed within the code and instead call out at the top or in a specific place. Feel free to propose changes to the spec to help clarify this

Reply:
Thanks for clarifying that. Since this PR already has 3 approvals, I'll merge it and submit a new one with this change as well as the one about checks that Gregory mentioned so it will be easier to review.